redis/src/commands/json.rs

```diff
@@ -0,0 +1,231 @@
+// can't use rustfmt here because it screws up the file.
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use crate::cmd::{cmd, Cmd};
+use crate::connection::ConnectionLike;
+use crate::pipeline::Pipeline;
+use crate::types::{FromRedisValue, RedisResult, ToRedisArgs};
+use crate::{RedisError, ErrorKind};
+
+#[cfg(feature = "cluster")]
+use crate::commands::ClusterPipeline;
+
+use serde::ser::Serialize;
+
+macro_rules! json_serialize_fail {
+	($err: expr) => {
+		Err(
+			RedisError::from((
+				ErrorKind::Serialize,
+				"Serialisation Error",
```

Comment:
Nit: please avoid UK spelling and use "serialization"/"serialize" everywhere.

Reply:
the word Serialise vs Serialize makes me dislike English even more.

It's my first language and it's just a pain lol